examples/large_files/test/tests.ts

```diff
@@ -1,233 +1,90 @@
-                    );
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        {
-            name: 'Unauthorized get hash status from actor',
-            test: async () => {
-                const actor = await createActor(
-                    getCanisterId('backend'),
-                    unauthorizedUser
-                );
-                try {
-                    await actor.get_hash_status('assets/test0B');
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        {
-            name: 'Unauthorized get hash from actor',
-            test: async () => {
-                const actor = await createActor(
-                    getCanisterId('backend'),
-                    unauthorizedUser
-                );
-                try {
-                    await actor.get_file_hash('assets/test0B');
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        {
-            name: 'Unauthorized clear file and info from actor',
-            test: async () => {
-                const actor = await createActor(
-                    getCanisterId('backend'),
-                    unauthorizedUser
-                );
-                try {
-                    await actor.clear_file_and_info('assets/test0B');
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        ...generateStandardFileTests('Upload', origin),
-        {
-            name: 'redeploy',
-            prep: async () => {
-                await generateTestFileOfSize(1, 'KiB');
-                await generateTestFileOfSize(10, 'KiB');
-                await generateTestFileOfSize(100, 'KiB');
-                execSync(`dfx deploy --upgrade-unchanged`, {
+    return () => {
+        beforeAll(async () => {
+            // Ensure all files from previous runs are cleared out
```

Comment:
My thoughts are that it would be nice to clear out the files after the test so they aren't clutter, but we should probably remove them before so that we are confident they aren't there (if the process was aborted before clean up) but it looks a tad redundant. Thoughts?

Reply:
We decided that both is find